sig-security-assessments/Documentation/recipe-book.md

```diff
@@ -0,0 +1,52 @@
+# Recipe Book 👩‍🍳
+🎉Yay! You have decided to do a Self-assessment and make your corner of Kubernetes more secure! We are so happy 
+to have you here. Below is an outline of the journey to get to a successful Self-assessment.
+
+## Preparation
+Set yourself up for success – build your team and get organized
+1.	Figure out who can do this Self-assessment with you. You will need a security expert and a counterpart who owns the 
+project and is an expert in it. If you are requesting a Security Self-assessment you can reach out to the team in
+[#sig-security](https://kubernetes.slack.com/messages/sig-security) to see if there is someone interested in helping to
+champion the assessment.
+1.	Make sure that you and your counterpart each have a back-up person assigned who attends all the meetings and is 
+involved in the assessment. Two heads are better than one, and redundancy means that if someone can’t make a meeting 
+for whatever reason, that you can still make progress.
+1.	To get further participation, advertise in other SIG meetings. Aim for a minimum of 4 people (a project expert, a 
```

Comment:
@aladewberry - is this intended for someone in the process of starting a Self-assessment (requester) or is this someone looking for projects that may want a Self-Assessment?